README.md

```diff
@@ -92,7 +92,9 @@ Firecracker's overall architecture is described in
 
 Firecracker consists of a single micro Virtual Machine Manager process that
 exposes an API endpoint to the host once started. The API is
-[specified in OpenAPI format](src/api_server/swagger/firecracker.yaml). Read more
+[specified in OpenAPI format](
+  src/firecracker/src/api_server/swagger/firecracker.yaml
```

Comment:
Mh, I think this one would have a better home at `src/firecracker/swagger`, instead of hiding it inside the module

Reply:
Or just under `docs/swagger` maybe? It's quite weird that its in the source code folder in the first place tbh (but leaving this up to you)

Reply:
moved it into `src/firecracker/swagger/`. It could go into docs, but that is up to you

Reply:
Let's preserve the status quo for now then and leave it in src :)